### ISSUE TITLE
Trips sorting order

### ISSUE THREAD

**Issue by hkvlhm** [BlinkTagInc/gtfs-to-html#32]
We have an issue when a trip begins at a stop in the middle of the timetable's stop order.

The timetables have trips sorted by the first stop time, without taking into account which stop is first for each trip. This results in some rows showing times out of order as they're read left to right. See below at "HSU Library Circle" and rows below that.

We propose that the trips be sorted by the stop time of the first stop every trip has in common. For example, sorting the trips shown below by the stop times at "HSU Library Circle".

![screen shot 2017-06-15 at 6 10 09 pm](https://user-images.githubusercontent.com/22775403/27207761-ea542c00-51f5-11e7-8533-9aae37b2c153.png)

**Comment by antrim** [comment]
@brendannee I thought it might be useful to have the GTFS/agency context here.

The example above is Redwood Transit System (http://redwoodtransit.org/) "Mainline" route.

GTFS: 
* https://transit.land/dispatcher/feeds/f-9pp-humboldtcounty~ca~us
* http://data.trilliumtransit.com/gtfs/humboldtcounty-ca-us/

**Comment by brendannee** [comment]
I just pushed an update that attempts to sort by departure_time at the first stop that all trips have in common.

If there are no stops in common across trips (thats a pretty dumb route in my opinion) then it defaults to sorting by the first stoptime in each trip).

I tried it with the redwood transit timetable_id=17 and it looks like its working.

Check it out out and let me know if its working as expected.
